stylesheet/coc.css

```diff
@@ -88,11 +86,58 @@ main {
 /* ---------Footer-------- */
 
 .page__footer {
-  text-align: center;
+  display: flex;
+  width: 100%;
+  color: #ffffff;
+  background-color: #404040;
+  justify-content: center;
+}
+
+.footer-content {
+  display: grid;
   width: 100%;
-  padding: 1em;
+  max-width: 40em;
+  padding: 1.5em 2em;
+  border-top: 1px solid #cccccc;
+  grid-template-columns: 1fr 1fr 1fr;
```

Comment:
@notlmn try to fix this nasty thing, add fallback layout other than grid if you can